1847 improve handling when djdt views dont respond with json (#1877)
jazzband/django-debug-toolbar: docs/changes.rst

Pending
-------

> * Display a better error message when request toolbar are not a valid json

Reviewer: ```suggestion
* Display a better error message when the toolbar's requests
  return invalid json.
```
